src/doc_builder/convert_rst_to_mdx.py

```diff
@@ -431,8 +431,10 @@ def parse_rst_docstring(docstring):
 
         # Returns section
         elif _re_returns.search(lines[idx]) is not None:
+            # tag is either `return` or `yield`
```

Comment:
Could we group the `Raises` block with this? We would just need a map from `tag` to `tag_type` that sends `raises` to `rasiederrors` and `returns`/`yields` to `returnstype`/`yieldstype`.